templates/environment/cf/cfn-execution-role.yml

```diff
@@ -0,0 +1,39 @@
+CloudformationExecutionRole:
+  Type: AWS::IAM::Role
+{{- if not .ImportVpc}}
+  DependsOn: VPC
+{{- end}}
+  Properties:
+    RoleName: !Sub ${AWS::StackName}-CFNExecutionRole
+    AssumeRolePolicyDocument:
+      Version: '2012-10-17'
+      Statement:
+      - Effect: Allow
+        Principal:
+          Service:
+          - 'cloudformation.amazonaws.com'
+          - 'lambda.amazonaws.com'
+        Action: sts:AssumeRole
+    Path: /
+    Policies:
+      - PolicyName: executeCfn
+        # This policy is more permissive than the managed PowerUserAccess
+        # since it allows arbitrary role creation, which is needed for the
+        # ECS task role specified by the customers.
+        # TODO: #335 Scope down the permissions given the CFN execution
```

Comment:
Can we get rid of this since the issue iz closed? And we'll allow configuring a scopedown policy at some point.